internal/runtime/hcsv2/container.go

```diff
@@ -116,6 +118,16 @@ func (c *Container) Kill(ctx context.Context, signal syscall.Signal) error {
 	return nil
 }
 
+func (c *Container) DeleteContainerState(ctx context.Context) error {
+	if c.isSandbox {
+		// remove user mounts in sandbox container
+		if err := storage.UnmountAllInPath(ctx, getSandboxMountsDir(c.id), true); err != nil {
```

Comment:
Should this be best effort and just log the error? IE: If unmount fails, we should probably still tear down the container right?